inlong-manager/manager-dao/src/main/resources/mappers/StreamSinkEntityMapper.xml

```diff
@@ -370,7 +370,7 @@
     </select>
     <select id="selectAllStreams" resultType="org.apache.inlong.manager.pojo.sort.standalone.SortSourceStreamSinkInfo">
         select inlong_cluster_name as sortClusterName,
-               sort_task_name,
+               data_node_name,
```

Comment:
This SQL need sort_task_name and data_node_name.
sort_task_name is a name of SortTask.
A sortTask has many stream sink, every stream sink has a data node.

Reply:
fix, use sort task name